__tests__/sass/mdn/_button-states-mixins-test.scss

```diff
@@ -0,0 +1,16 @@
+@include describe('active-button-state') {
```

Comment:
Why is ``__tests__ `` (which looks super ugly but apparently is a React best practice?) a top-level folder?

Reply:
For KumaScript, I found that the other Jest convention was to name the tests as *.test.js, and it will find them even if they aren't in the ugly __tests__ folder. So that is something we could go with as well.

Reply:
Separate PR sounds right to me.